driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/layout/LayoutCheck.java

```diff
@@ -72,6 +72,7 @@ public enum Properties implements IProperties {
         ACTUAL_PATH("actual.path", "src/test/resources/screenreferences/actual"),
         USE_IGNORE_COLOR("use.ignore.color", false),
         PIXEL_RGB_DEVIATION_PERCENT("pixel.rgb.deviation.percent", 0.0),
+        PIXEL_COUNT_ASSERTION("pixel.count.assertion", "optional"),
```

Comment:
Like discussed the property should converted to a bool property and could called `tt.layoutcheck.pixel.count.hard.assertion`.
Default value is `false`